src/components/tab-group/bl-tab-group.css

```diff
@@ -0,0 +1,15 @@
+.tabs {
+  background-color: var(--bl-color-primary-background);
+  border-bottom: 2px solid var(--bl-color-secondary-background);
+  display: flex;
+  flex-direction: row;
+}
+
+.tabs-list {
+  overflow-x: scroll;
+  border-radius: 6px 6px 0 0;
+}
+
+.panels {
+  border-radius: 0 0 6px 6px;
```

Comment:
I know we don't have a variable for `6px`. We'll have some new variables for border-radius with #185 Please can you add `FIXME` comments here to use variables once they are implemented